core/src/main/java/nl/nn/adapterframework/doc/FrankDocJsonFactory.java

```diff
@@ -0,0 +1,124 @@
+package nl.nn.adapterframework.doc;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.Logger;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
```

Comment:
Wij willen graag van `org.json` af, in plaats hiervan gebruiken wij overal in het framework de `javax.json` variant.

Reply:
Gedaan.